ITISFoundation/osparc-simcore
# 🎨🔒️ Remove deprecated services for the SPARC Portal

## What do these changes do?
Adds a filter on the list returned by `/v0/services` to remove deprecated services (the ones that have an entry in DB, in the table `services_meta_data`, column `deprecated`.


## Related issue/s
- This PR follows up on  https://github.com/ITISFoundation/osparc-simcore/pull/4024
- resolves ITISFoundation/osparc-issues#683



## How to test

Deploy the stack, go to `/v0/services`, in that list services that have a deprecation date in the DB, should not appear.

## How did I test it?
I've exported and imported services data from osparc.io DB into my local deploy DB (applying as filter `jupyter`). 
Before applying this PR, going to `/v0/services` in the browser gave 10 results. After applying the PR `/v0/services` gave 5 results. I then double-checked that the 5 missing ones are deprecated ones. 

## DevOps Checklist
